java/memory/src/main/java/io/netty/buffer/ArrowBuf.java

```diff
@@ -109,8 +109,8 @@ public ArrowBuf reallocIfNeeded(final int size) {
     if (bufManager != null) {
       return bufManager.replace(this, size);
     } else {
-      throw new UnsupportedOperationException("Realloc is only available in the context of an " +
-          "operator's UDFs");
+      throw new UnsupportedOperationException("Realloc is only available in the context of an "
+          + "operator's UDFs");
```

Comment:
Our current checkstyle rule for `OperatorWrapCheck` wants the operator on a new line like this. Doesn't matter to me, but I am more used to having it at the end of the line, which seems to be how it mostly is already. Thoughts @jacques-n or @icexelloss ?

Reply:
I am used to having it at end of line as well. So light +1 on that.

Reply:
slight +1 for me too, so I will proceed with having operators on the end of the line